api/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for hs-opentelemetry-api
 
+## 0.2.1.0
+
+- defined and exported `toImmutableSpan` and `FrozenOrDropped` from `OpenTelemetry.Trace.Core`
+
```

Comment:
nit: this feels like a patch more than a minor version bump

Reply:
I'm following this guide

> Non-breaking change. Otherwise, if only new bindings, types, classes, non-orphan instances or modules (but see below) were added to the interface, then A.B MAY remain the same but the new C MUST be greater than the old C. Note that modifying imports or depending on a newer version of another package may cause extra non-orphan instances to be exported and thus force a minor version change.

https://pvp.haskell.org/

Idk if that's the scheme adhered by this project though.

Reply:
the PVP is weird in that it's `MAJOR.MAJOR.MINOR.PATCH` where you can omit the leading `MAJOR` if you choose to do so.

the current version is `0.2.0.0`, so `0.2.1.0` is a minor bump, whereas in the example on that page `0.2.0` -> `0.2.1` would be a patch bump.